packages/client/src/components/Header/Header.tsx

```diff
@@ -365,9 +365,7 @@ class HeaderComp extends React.Component<IFullProps, IState> {
       (NATL_ADMIN_ROLES.includes(role as string) &&
         location.includes(PERFORMANCE_HOME)) ||
       (SYS_ADMIN_ROLES.includes(role as string) &&
-        location.includes(PERFORMANCE_HOME)) ||
-      (REGISTRAR_ROLES.includes(role as string) &&
-        location.includes(REGISTRAR_HOME))
```

Comment:
Is this change necessary? Because it might again cause issues with the history buttons